.github/workflows/test.yml

```
@@ -11,8 +11,17 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  feature-matrix:
+    uses: ./.github/workflows/feature-matrix.yaml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/feature-matrix.yaml": open /home/jailuser/git/.github/workflows/feature-matrix.yaml: no such file or directory

(workflow-call)